src/bootstrap/compile.rs

```diff
@@ -166,7 +161,12 @@ fn copy_self_contained_objects(
     compiler: &Compiler,
     target: Interned<String>,
 ) -> Vec<(PathBuf, DependencyType)> {
-    let libdir = builder.sysroot_libdir(*compiler, target);
+    let libdir_self_contained = if compiler.stage == 0 {
```

Comment:
This will need to be removed during the next bootstrap compiler update?
It's easy to miss, this should probably be mentioned explicitly in a commend (with `cfg(bootstrap)` as a part of the comment for searchability).